elections/committee-conflict-of-interest.md

```diff
@@ -0,0 +1,57 @@
+# Committee Conflicts of Interest
+
+At time of writing, the only known conflict of interest for serving on
+committees in the Kubernetes Community exists with the
+[Steering Committee](/committee-steering/README.md) and
+the [Code of Conduct Committee (CoCC)](/committee-code-of-conduct/README.md).
+
+## Steering Committee and Code of Conduct Committee
+
+**_Kubernetes Community members cannot serve on both committees simultaneously._**
+
+This conflict may arise in the following scenarios:
+
+- A standing Steering Committee member has accepted a nomination to serve on
+  the Code of Conduct Committee
+- A standing CoCC member has elected to run for a seat on the Steering
+  Committee
+- A Kubernetes Community member has been nominated for candidacy for both
+  committees
+
+There are several factors under consideration when formulating a reasonable
+process for resolving this conflict of interest:
+
+- The distinctness of the Steering and CoCC charters, not limited to how these
+  committees need to adjudicate on community matters
+- The Kubernetes Community's expectation of commitment in its committee members
+  for duration of their terms
+- Continuity of committees and the duties they discharge
+- Undue burden on existing committee members to balance workload through
+  committee membership transitions, not limited to onboarding/offboarding
+- Undue burden on Election Officers in supporting overlapping elections with
+  the potential to generate conflicts
+- Minimization of conflict of interest edge cases not otherwise described by
+  this document
+
+With these considerations in mind, the following rules exist to prevent or
+resolve membership conflicts of interest between Steering and the CoCC:
+
+<!-- TODO(committee-coi): Reflect this list in each committee's "Eligibility for candidacy" section -->
+
+- Loss of committee member that occurs due to this conflict of interest will
+  be resolved per existing election or committee vacancy rules
+- Candidates who are nominated for a seat on both Steering and CoCC within the
+  same 12-month period must:
+  - (if elected) accept a seat on the committee whose election concludes first
+  - be withdrawn from candidacy in second election to conclude
+- Standing Steering or CoCC members must have served on their committee for a
+  period of twelve (12) months to be considered eligible for candidacy in the
+  other committee's election
+- Kubernetes Community members who have recently vacated a seat on either
+  Steering or CoCC will be ineligible for candidacy for a period of three (3)
+  months
```

Comment:
Reading this bullet over, I wonder if it could be interpreted as disallowing a committee member who is about to complete their term from running for re-election.

If others are interpreting it that way, I'm open to edit suggestions. :)

Reply:
This should clear up any potential confusion over your intent here

```suggestion
- Kubernetes Community members who have recently voluntarily vacated a seat on either
  Steering or CoCC without serving the complete term will be ineligible for candidacy for a period of three (3)
  months
```

Reply:
I think the edit from @katcosgrove provides good clarification and captures the intent. 👍

Reply:
Neato; thanks Kat!